frontend/components/UX/GlobalButtons.vue

```diff
@@ -8,31 +8,15 @@
   >
     <v-divider v-if="viewConfig.dividers.before" />
 
-    <v-row :class="`${viewConfig.btnsRowClass} ${isMobileWidth ? 'my-0' : ''}`">
+    <v-row 
+      :class="`${viewConfig.btnsRowClass} ${isMobileWidth ? viewConfig.btnsRowClassMobile : ''}`"
+      >
       <div
         v-for="(btn, index) in viewConfig.componentButtons"
-        :id="'B' + btn.id + index"
-        :key="'B' + btn.id + index"
+        :id="'B-' + btn.id + '-' + index"
+        :key="'B-' + btn.id + '-' + index"
```

Comment:
Template strings are nice for this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals